.github/workflows/build.yml

```diff
@@ -0,0 +1,80 @@
+name: Tag, Build, and Push Image
+
+on:
+  push:
+    branches:
+    - dev
+    paths-ignore:
+    - 'README.md'
+  pull_request:
+    branches:
+    - dev
+    paths-ignore:
+    - 'README.md'
+
+env:
+  REGISTRY_HOST: gcr.io
+  GOOGLE_PROJECT: broad-dsp-gcr-public
+  SERVICE_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  tag-build-push:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Get Short Sha
+      id: short-sha
+      run: echo "::set-output name=sha::$(git rev-parse --short=12 HEAD)"
+    - name: Auth to GCR
+      uses: google-github-actions/setup-gcloud@master
+      with:
+        version: '270.0.0'
+        service_account_email: ${{ secrets.GCR_PUBLISH_EMAIL }}
```

Comment:
@mflinn-broad @choover-broad I essentially copied this from https://github.com/DataBiosphere/duos-ui. Is there anything we need to do to add secrets to this repo? I think the secrets this workflow references are:
- `GCR_PUBLISH_EMAIL`
- `GCR_PUBLISH_KEY`
- `REPO_ACCESS_TOKEN`
- `SLACK_WEBHOOK_URL`
- `GITHUB_TOKEN`

Reply:
Those secrets are all stored in vault, I can add them to this repo for you, a few of them are only accessible to devops

Reply:
Thanks @mflinn-broad!

Reply:
@rtitle should be all set. I used `BROADBOT_TOKEN` instead of `REPO_ACCESS_TOKEN`. I think it makes it more clear what that secret values actually is. Do you mind updating references to that value?

Reply:
Done, and the GCR push worked! (All github actions passed on this PR.) Still working on the terra-k8s infrastructure PRs.